po/de.po

```diff
@@ -10116,15 +10139,17 @@ msgstr "Setze Zoom auf %.0f"
 msgid "Cannot Build. Oil Resource Burning."
 msgstr "Bau nicht möglich. Ölquelle brennt."
 
+# Ich habe im folgenden Hitpoints durchgehend mit Trefferpunkte übersetzt. Ist an einigen Stellen
+# auch von Dir so übersetzt worden und ergibt mehr Sinn in meinen Augen. -alfred007
```

Comment:
Leider hat sich das WZP ja dagegen entschieden den Spirit des Originalspiels zu erhalten, in dem das noch "Rumpfpunkte" waren. ;)

Passt!